src/main/java/qna/domain/Answer.java

```diff
@@ -3,13 +3,20 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
+@Entity
+public class Answer extends  BaseEntity {
 
```

Comment:
```suggestion
import javax.persistence.*;
import java.util.Objects;

@Entity
public class Answer extends BaseEntity {
```
가독성을  위해 개행을 적절히 해주세요 :)